core-ui/src/components/Predefined/Details/Service/Service.details.js

```diff
@@ -90,6 +96,16 @@ export const ServicesDetails = ({ DefaultRenderer, ...otherParams }) => {
         </>
       ),
     },
+    {
+      header: t('services.external-ips'),
+      value: service => (
+        <ul>
+          {service.spec.externalIPs?.map(ip => (
+            <li key={ip}>{ip}</li>
+          ))}
+        </ul>
```

Comment:
Nitpicking time! When the value is empty, we use '-' as a placeholder, so please use it in `externalIPs` field on both details and list.

Reply:
Example - when there are no labels.